lib/sitemap_generator/interpreter.rb

```diff
@@ -9,6 +9,10 @@ class Interpreter
     if SitemapGenerator.app.is_at_least_rails3?
       if !::Rails.application.nil?
         include ::Rails.application.routes.url_helpers
+        unless ::Webpacker.instance.nil?
```

Comment:
`Webpacker` will cause a NameError if it is not defined.  It's probably easier for you to dynamically include your helpers rather than modify the gem.

```
SitemapGenerator::Interpreter.send(:include, ActionView::Helpers::AssetUrlHelper)
SitemapGenerator::Interpreter.send(:include, Webpacker::Helper)
```
or
```
module SitemapGenerator
  class Interpreter
    include ::ActionView::Helpers::AssetUrlHelper
    include ::Webpacker::Helper
  end
end
```

That could be at the top of your sitemap config or in a Rails initializer.